linera-views/Cargo.toml

```diff
@@ -58,6 +58,9 @@ tokio = { workspace = true, features = ["rt", "sync"] }
 tracing.workspace = true
 trait-variant.workspace = true
 
+[target.'cfg(not(target_arch = "wasm32"))'.dependencies]
+criterion = { workspace = true, features = ["async_tokio"] }
```

Comment:
Should this be optional and controlled by a test features?

Reply:
I agree. My vote goes to using the `test` feature rather than adding a new one.

Reply:
Yes, I did that.
But, I still have the `criterion` in the `Cargo.lock` of the examples which is unfortunate.